osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -36,6 +36,8 @@ protected override void ParseStreamInto(LineBufferedReader stream, T output)
                 if (ShouldSkipLine(line))
                     continue;
 
+                line = line.Trim();
```

Comment:
I'm fairly certain this should be `line = StripComments(line).TrimEnd()` if anything. Especially don't trim the start - that breaks storyboard decoding.

Also, can remove calls to `StripComments()` from other areas.

Reply:
Right, didn't realise this was also being used to parse storyboard content as well.